tool.go

```diff
@@ -24,6 +24,9 @@ type Context struct {
 // The path "std" is expanded to all packages in the Go standard library.
 // The path "cmd" is expanded to all Go standard commands.
 // The string "..." is treated as a wildcard within a path.
+// When matching using wilcards, directories with the prefix of dot or
```

Comment:
Agreed on all accounts, but if we drop the wording of wildcards, would someone think a path of `./testdata` would be excluded (it wouldn't be)?

Perhaps "when recursively matching" or "when matching multiple packages"?

```
// When matching recursively, directories are ignored if they are prefixed with a
// dot or an underscore (such as ".foo" or "_foo"), or are named "testdata".
```